stories/generic-tile/generic-tile.stories.js

```diff
@@ -25,148 +74,155 @@ export const genericTileSizes = () => `<div class="fddocs-container">
     </div>
 </div>`;
 
+genericTileSizes.storyName = 'Sizes';
+
 genericTileSizes.parameters = {
     docs: {
         iframeHeight: 500,
         storyDescription: `
-The large (default) tiles are designed for screens larger than 374px. For smaller screens use the small tiles \`fd-tile--s\`.
--   Large Tile - 11 x 11rem
--   Small Tile - 9.25 x 9.25rem
+Generic tile can be displayed in two sizes, depending on the screen width.
+
+Size | Modifier class | rem | Screen width
+:----| :------------- | :-- | :----------- 
+Small | \`fd-tile--s\` | 9.25 x 9.25rem | > 374 px
+Large | \`fd-tile--l\` | 11 x 11rem | >= 374 px
         ` }
 };
 
-export const genericTileLayout = () => `<div class="fddocs-container">
+export const Layout = () => `<div class="fddocs-container">
     <div role="button" aria-label="button" tabindex="0" class="fd-tile">
-        <div class="fd-tile__header" style="background-color: #C0E0FA">
+        <div class="fd-tile__header fddocs-tile-example-border">
             <div class="fd-tile__title">.fd-tile__title</div>
             <div class="fd-tile__subtitle">.fd-tile__subtitle</div>
         </div>
-        <div class="fd-tile__content" style="background-color: #C0E0FA">.fd-tile__content</div>
-        <div class="fd-tile__footer" style="background-color: #C0E0FA">.fd-tile__footer</div>
+        <div class="fd-tile__content fddocs-tile-example-border">.fd-tile__content</div>
+        <div class="fd-tile__footer fddocs-tile-example-border">.fd-tile__footer</div>
     </div>
     <div role="button" aria-label="button" tabindex="0" class="fd-tile fd-tile--double">
-        <div class="fd-tile__header" style="background-color: #C0E0FA">
+        <div class="fd-tile__header fddocs-tile-example-border">
             <div class="fd-tile__title">.fd-tile__title</div>
             <div class="fd-tile__subtitle">.fd-tile__subtitle</div>
         </div>
-        <div class="fd-tile__content" style="background-color: #C0E0FA">.fd-tile__content</div>
-        <div class="fd-tile__footer" style="background-color: #C0E0FA">.fd-tile__footer</div>
+        <div class="fd-tile__content fddocs-tile-example-border">.fd-tile__content</div>
```

Comment:
Just FYI - styles from this class are only in canvas on local version of storybook